addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4853,6 +4853,13 @@ registry.SnippetSave = SnippetOptionWidget.extend({
                                 const snippetName = dialog.el.querySelector('.o_we_snippet_name_input').value;
                                 const targetCopyEl = this.$target[0].cloneNode(true);
                                 delete targetCopyEl.dataset.name;
+                                // Removing the ids is especially important for tabs snippet which
+                                // uses anchor. Indeed, it is important that the elements of the 
+                                // saved tabs located on the editor and the elements of the tabs 
+                                // on the webpage do not reffer to the same ids.
+                                for (const tabIDTargetCopyEl of targetCopyEl.querySelectorAll('[id]')) {
+                                    tabIDTargetCopyEl.id = "";
+                                }
```

Comment:
- Commit message: opw-XXX instead of task-XXX since it is a customer ticket and not a normal task.
- Comment here: is not that "refer" instead of "reffer"? (could be nice to add a spelling checker to your editor if possible (I use "Code Spell Checker" in VS code)
- In the end, I wonder: the normal s_tabs.xml definition contains IDs for nav-link and tab-pane as well so could create the bug you describe (I guess not really because the one in the page are completed with some random number at the end to prevent the issue). As a stable fix, maybe the generation of the right panel snippets could automatically strip IDs on nav-link and tab-pane too so they don't conflict with the page? (that way we would also solve the problem for existing tabs snippet users already saved (and so the ticket's customer one)). So:
    - In stable + master: what you did is to be done, good (removing ID's in saved snippets)
    - In stable at least (so something to add to this PR): also strip ID from tab-pane and nav-link in custom snippets when generating the right panel (`computeSnippetTemplates`) -> that way you fix user issues without them having to re-save their tab snippet.